include/z64player.h

```diff
@@ -726,7 +726,7 @@ typedef struct Player {
     /* 0x0445 */ u8 prevCsAction;
     /* 0x0446 */ u8 cueId;
     /* 0x0447 */ u8 unk_447;
-    /* 0x0448 */ Actor* unk_448;
+    /* 0x0448 */ Actor* csActor; // Actor involved in a `csAction`. Typically the actor that invoked the cutscene.
```

Comment:
```suggestion
    /* 0x0448 */ Actor* csActor; // Actor involved in a `csAction`. Typically the actor that invoked the cutscene or NULL.
```
?

Reply:
I suppose. my comment is mainly concerning when it is actually set, not what it is typically set to in general

Reply:
I can change it to be one sentence so its clear what i mean by "typically": 
`// Actor involved in a `csAction`, typically the actor that invoked the cutscene.`
I still dont think describing NULL is needed, Im just saying what the field represents when its relevant